src/main/java/model/ticket/Ticket.java

```diff
@@ -115,4 +132,25 @@ public String toString() {
                 ", price=" + price +
                 '}';
     }
-}
\ No newline at end of file
+
+    @Override
+    public void print() {
+        System.out.println("Class content: " + this);
+    }
+
+    public void share() {
+        System.out.println("Default share");
+    }
+
+    public void share(String phone) {
+        System.out.println("Default share by phone");
+    }
+
+    public void share(String phone, String email) {
+        System.out.println("Default share by phone and email");
+    }
```

Comment:
good, just add @Override for each method

Reply:
Added missed annotations